daemon/src/maker_cfd.rs

```diff
@@ -765,18 +770,19 @@ where
             .address()
             .expect("actor to be able to give address to itself");
 
+        let task = async move {
+            let dlc = contract_future.await;
+
+            log_error!(this.do_send_async(CfdRollOverCompleted { order_id, dlc }));
```

Comment:
I don't think we should `log_error` here, this only ever fails with `Disconnected` which is unlikely (impossible?) because we are sending the message to ourselves.

Reply:
it was possible to get this one if the actor got shut down whilst doing contract setup (which takes ~1 min). With the guard that I've added here, it shouldn't ever happen, so I'm happy to do an `.expect("actor to be connected")` here. 

I'm not fussed, I just didn't want to swallow the error silently, and I also didn't want to open the doors for anything else than `RemoteHandle<()>`

Reply:
used `expect()` (as the master branch does now too)